AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -29,6 +29,7 @@ public final class CardComponent: PaymentComponent, PresentableComponent, Locali
     /// :nodoc:
     internal var cardTypeProvider: AnyCardTypeProvider
     
+    private var topElements: [CardType]
```

Comment:
can we have a more descriptive name? `topElements` can be any thing.

Reply:
updated